readers-guide.md

```diff
@@ -1,6 +1,7 @@
 # Readers guide
 
-The criteria in the Standard have consistent sections that together make it clear how to create great public code.
+The Standard describes a number of criteria. Each criteria have consistent sections that make it clear how to create great public code.
+Below is a brief explanation of each of these sections and how they are used within the criteria of the Standard.
```

Comment:
* Singular of criteria is criterion. So either we do 'Each criterion has' or 'All criteria have'
* Inconsistent new line behaviour.

```suggestion
The Standard describes a number of criteria.
All criteria have consistent sections that make it clear how to create great public code.
Below is a brief explanation of each of these sections and how they are used within the criteria of the Standard.
```